Summary: headers under include/qwt-qt<major>, so the Qt 5 build installs beside the Qt 6 one
----

--- a/qwtconfig.pri
+++ b/qwtconfig.pri
@@ -29,7 +29,7 @@
 }
 
 QWT_INSTALL_DOCS      = $${QWT_INSTALL_PREFIX}/share/doc/qwt
-QWT_INSTALL_HEADERS   = $${QWT_INSTALL_PREFIX}/include/qwt
+QWT_INSTALL_HEADERS   = $${QWT_INSTALL_PREFIX}/include/qwt-qt$${QT_MAJOR_VERSION}
 QWT_INSTALL_LIBS      = $${QWT_INSTALL_PREFIX}/lib
 
 ######################################################################
